modules/crawler.py

```diff
@@ -175,39 +162,33 @@ def get_tracert(ip_address):
     output, _ = tracert.communicate()
     return output.decode().strip().split("\n")
 
-
-def parse_classes(soup):
-    tag_indicators = []
+@return_empty_if_fails
+def parse_classes(soup) -> list[Indicator]:
     used_classes = set()
     for elem in soup.select("[class]"):
         classes = elem["class"]
         used_classes.update(classes)
-    tag_indicators.append(add_indicator("3-css_classes", list(used_classes)))
-    return tag_indicators
+    if len(used_classes) > 0:
+        return [Indicator("3-css_classes", list(used_classes))]
+    return []
```

Comment:
before, if `used_classes` was empty, we returned an empty list as an indicator. now we don't